vendor/github.com/go-openapi/validate/schema_option.go

```diff
@@ -14,20 +14,41 @@
 
 package validate
 
+// SchemaValidatorOptions defines optional rules for schema validation
 type SchemaValidatorOptions struct {
-	DisableObjectArrayTypeCheck bool
+	EnableObjectArrayTypeCheck    bool
+	EnableArrayMustHaveItemsCheck bool
```

Comment:
do we need to turn these on when validating the schema itself? does this change to default these off loosen our validation on CRD's themselves (not the CRs)?

Reply:
We don’t use meta validation of schemas using go-openapi.